accname/name/comp_text_node.html

```diff
@@ -13,23 +13,96 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_text_node">#comp_text_node</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
```

Comment:
Having trouble nailing down what constitutes a "text node"; in the spec, it is capitalized as "Text" so perhaps it refers to the DOM "Text" interface? https://dom.spec.whatwg.org/#interface-text

I understood this step of the comp algorithm as testing nodes that are non-interactive and only contain text (e.g., headings, paragraph, caption). Do we need greater clarification on what a "text node" is?

Reply:
These are inherited from XML terms: most commonly _text node_ versus _element node_ versus _comment node_. Element nodes can contain other element, comment, and text nodes, but IIRC text nodes are always leaf nodes.

Reply:
> I initially understood this step of the comp algorithm as testing nodes that are non-interactive and only contain text (e.g., headings, paragraph, caption), or text as children nodes of HTML tags. 

hence testing the text nodes split by comment nodes, with or without additional spaces

```html
<h3 class="ex" data-expectedlabel="headinglabel" data-testname="heading with text/comment/text nodes, no space">
  heading<!-- with non-text node splitting concatenated text nodes -->label<!-- [sic] no extra spaces around first comment -->
</h3>
```

Reply:
Maybe your comment is that AccName should link to specific portion of the DOM spec? If so, +1

Reply:
Created new AccName issue here: https://github.com/w3c/accname/issues/212